### PR TITLE
Add cost estimates dk

### PR DESCRIPTION
**Pull request by DinneK**
#251 
## Problem
Need a front-end visualization of estimated costs of developing a project.

## Solution
Create a bar graph using Pygal to implement the missing field for cost estimates.

## Result
A bar chart depicting cost estimates populates in the browser.

## Test Plan
Manually test in the browser.

Screenshot 2024-10-10 at 6 30 41 PM

### CODE REVIEW

File: scripts/gen_graphs.py [hunk 1]
```diff
@@ -313,3 +315,59 @@ def generate_dryness_percentage_graph(oss_entity):
...
+
+def generate_cost_estimates_bar_chart(oss_entity):
+    """
+    This function generates a pygal bar chart for estimated costs with rounded values and a dollar sign.
+
+    Arguments:
+        oss_entity: the OSSEntity to create a graph for.
+    """
+
+    bar_chart = pygal.Bar()
```
[the remark this answers is not in the document]
Comment (DinneK) [reply scripts/gen_graphs.py]: Legend moved to bottom

File: scripts/gen_graphs.py [hunk 2]
```diff
@@ -313,3 +315,59 @@ def generate_dryness_percentage_graph(oss_entity):
...
+    metric_data = oss_entity.metric_data['cocomo']
+
+    estimatedCost_low = metric_data.get('estimatedCost_low', 0)
+    estimatedCost_high = metric_data.get('estimatedCost_high', 0)
+
+    bar_chart.value_formatter = lambda x: f'${x:,.2f}'
+
+    average_cost = (estimatedCost_low + estimatedCost_high) / 2
+
+    bar_chart.title = f'Estimated Project Costs in $ (Average Cost: ${average_cost:,.2f})'
```
[the remark this answers is not in the document]
Comment (DinneK) [reply scripts/gen_graphs.py]: The title now includes Average and COCOMO

Screenshot 2024-10-11 at 2 42 06 PM

**Review by IsaacMilarky — APPROVED** [review]
LGTM :+1:

**Review by DinneK — COMMENTED** [review]
Resolving merge conflicts

**Pull request merged**